{"name": "test_login_error[logins_data3]", "status": "passed", "attachments": [{"name": "log", "source": "7f8ed7e7-4789-42e9-ba8e-a7b14b3021e9-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "logins_data", "value": "{'username': 'yang', 'password': 123456, 'code': 1234, 'des': '验证码错误', 'expect': '验证码输入错误!', 'result': 300}"}], "start": 1596194769631, "stop": 1596194770349, "uuid": "7a68aa73-5293-4afb-b605-bcd1d1c363d3", "historyId": "397928278f182cca9f41beffbf4abdaa", "testCaseId": "d696b25ca08475d04e305c9ec05fe006", "fullName": "TestCase.test_login.TestLogin#test_login_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('in_web', 'logins_data')"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-F7KFRCJP"}, {"name": "thread", "value": "9200-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_login"}]}